docs/intro/tutorial05.txt

```diff
@@ -162,7 +162,7 @@ what we can do in an automated test, so let's turn that into an automated test.
 
 A conventional place for an application's tests is in the application's
 ``tests.py`` file; the testing system will automatically find tests in any file
-whose name begins with ``test``.
+whose name begins with ``test`` (function names in ``TestCase`` classes must begin with ``test``).
```

Comment:
This paragraph is about finding test **files**. 

That individual test methods must be named `test_` may be a point worth making but it would be better in the text following the `QuestionModelTests` example just below here.